### PR TITLE
Update run-applio.sh Fix macOS

### PR DESCRIPTION
**Pull request by MarxAnthony**
Screenshot 2024-12-23 at 5 44 38 PM
Screenshot 2024-12-23 at 5 51 54 PM

### CODE REVIEW

**Comment by Marxanthonydev** [comment]
Ahora se puede hacer inferencia mas rapido y se soluciona el error donde python se crashea y la cpu no se utiliza y cierra

File: run-applio.sh [hunk 1]
```diff
@@ -1,9 +1,10 @@
-#!/bin/sh
+\#!/bin/sh
 printf "\033]0;Applio\007"
 . .venv/bin/activate
 
  export PYTORCH_ENABLE_MPS_FALLBACK=1
  export PYTORCH_MPS_HIGH_WATERMARK_RATIO=0.0
+ export OMP_NUM_THREADS=8
```
Comment (blaisewf) [inline run-applio.sh:7]: what if someone has +8 threads?

Comment (Marxanthonydev) [reply run-applio.sh:7]: Well, I was faster with 8 in inference.

**Comment by Marxanthonydev** [comment]
I don't know whether to leave it at 8 or leave it at 1

**Comment by blaisewf** [comment]
When export `OMP_NUM_THREADS=1` is set, programs that use OpenMP for parallel processing will run with only one thread, even if the system has multiple cores or CPUs available.

**Comment by MarxAnthony** [comment]
The best option is to use export OMP_NUM_THREADS= so that the format is preset and the thread is system dependent

**Comment by blaisewf** [comment]
"The best option is to use export OMP_NUM_THREADS=1" but you changed to 2? I'm going to close this PR, this depends on each system, so let's leave to each user the decision to use the value they want.

{5A44E11B-C60C-4310-B6D8-A3DCD23D1D3F}

**Pull request closed without merging**